release_windows.bat

```diff
@@ -125,11 +125,6 @@ echo ****** check for and update outdated modules  ******
 echo ****************************************************
 for /F "skip=2 delims= " %%i in ('pip list --outdated') do py -m pip install --upgrade %%i
 
-rem echo ****************************************************************************
-rem echo ****** Bug on wxPython 4.1.x workaround for Python 3.x and Windows 10 ******
-rem echo ****************************************************************************
-rem wxPython 4.2.0 is available. Snapshot version is not needed now. Hopefully.  #2022-08-05
-rem pip install -U --pre -f https://wxpython.org/Phoenix/snapshot-builds/ wxPython
 
 echo *************************************************************************
 echo ****** pyglet workaround, needs to be below 2.0 (isn't compatible) ******
```

Comment:
Same comment as for the cairosvg workaround below.

Reply:
Yea this look a bit awkward.
In line 126 I update automatically all existing modules to the latest versions so that everything is up-to-date as there are a lot of additional modules involved. 
```
echo ****************************************************
echo ****** check for and update outdated modules  ******
echo ****************************************************
for /F "skip=2 delims= " %%i in ('pip list --outdated') do py -m pip install --upgrade %%i
```
This unfortunately update the two restricted modules pyglet and cairosvg too, so I need to correct them to the older release.
I know, it's not elegant but do what is needed and I can kick this part if there is an solution ( ... and it's my reminder that there is something to check in the future) :)

Reply:
Ah Ok, fair enough. I totally missed that line. Issue solved then :)